nemo/collections/tts/torch/g2ps.py

```diff
@@ -12,6 +12,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# TODO (xueyang): deprecate this file since no other places import modules from here anymore. However,
+# TODO @xueyang: deprecate this file since no other places import modules from here anymore. However,
```

Comment:
I think it might come back into use once TN is moved. @ekmb will you be moving G2P to this file again?

Reply:
yeah, just leave a TODO just incase.